.github/workflows/release.yaml

```diff
@@ -0,0 +1,16 @@
+name: OAS release
```

Comment:
Is there a need for a separate GHA file? I think it would make sense to add this in as a stage in the azure script. That way you can add a dependency, i.e., only push/publish if builds/tests stages pass.

Reply:
All our other pypi codes use this GHA as I don't believe any of them use Azure, I think it's fine to not have this depend on tests passing etc because it's only run when we manually create a new release

Reply:
We do use Azure for e.g. baseclasses [here](https://github.com/mdolab/baseclasses/blob/main/.github/azure-pipelines.yaml#L25) and the sphinx theme. You're right that most of the repos use GHA though (and are completely separate in CI toolchain from Azure).

Reply:
Oh right, I didn't realise that, I'll follow the baseclasses example then